openmeetings-db/src/main/java/org/apache/openmeetings/db/dao/calendar/AppointmentDao.java

```diff
@@ -48,6 +33,14 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.transaction.annotation.Transactional;
 
+import javax.persistence.EntityManager;
+import javax.persistence.NoResultException;
+import javax.persistence.PersistenceContext;
+import javax.persistence.TypedQuery;
+import java.util.*;
+
+import static org.apache.openmeetings.util.OpenmeetingsVariables.webAppRootKey;
+
 @Transactional
 public class AppointmentDao {
```

Comment:
Pretty much, [here's the code for it](https://github.com/apache/openmeetings/pull/1/files/1dbd0405aeef8a0e98a5796b9f9aae7214eee263#diff-cadae26eaa36f643e3ebde22a1dd109dR237).